character-controller.js

```diff
@@ -696,8 +696,6 @@ class StatePlayer extends PlayerBase {
           }
         }
       }
-    } else {

```

Comment:
IMO avatar syncing should not be attempted if there is no avatar instance id to switch to.

What about else `throw`? This seems like a legitimate bug if we need to do this, so it's best to call it out and at least get a stack trace.